src/main/scala/com/databricks/labs/overwatch/utils/Upgrade.scala

```diff
@@ -15,8 +15,53 @@ import scala.concurrent.forkjoin.ForkJoinPool
 object Upgrade extends SparkSessionWrapper {
 
   private val logger: Logger = Logger.getLogger(this.getClass)
+  import spark.implicits._
 
-  case class UpgradeStatus(db: String, tbl: String, actionTake: Option[String], errorMsg: Option[String])
+  def upgrade0412(prodWorkspace: Workspace): Dataset[UpgradeReport] = {
```

Comment:
There is a schema change so a schema upgrade will be required if jobs_scope was run prior to 0.4.12.